tcf_website/views/index.py

```diff
@@ -11,9 +11,13 @@ def index(request):
     Redirect to landing page if user not authorized, otherwise show
     browse page.
     """
```

Comment:
If we're not redirecting anymore then this docstring should be changed

Reply:
True, done